man/visualSubsetRecode.Rd

```diff
@@ -0,0 +1,60 @@
+\name{visualSubsetRecode}
+\alias{visualSubsetRecode}
+\title{Inspect data subsets visually and recode instantaneously}
+\description{
+This function allows manual inspection and manual recoding of subsets of the data.}
```

Comment:
For me, from the description it is not clear what the function does. Subsets of which kind of data? item data? subsets according to person groups? or different booklets? A little more detailed description would be helpful.